src/NuGet.Core/NuGet.Packaging/Signing/TrustStore/CertificateBundleX509ChainFactory.cs

```diff
@@ -4,14 +4,19 @@
 #if NET5_0_OR_GREATER
 
 using System;
+using System.Globalization;
 using System.IO;
+using System.Linq;
 using System.Security.Cryptography;
 using System.Security.Cryptography.X509Certificates;
+using NuGet.Common;
 
 namespace NuGet.Packaging.Signing
 {
     internal abstract class CertificateBundleX509ChainFactory : IX509ChainFactory
     {
+        internal const string NU3042HelpUrl = "https://aka.ms/nuget/nu3042";
```

Comment:
I think if the url is NU3042, then we shouldn't need it.

Reply:
> I think if the url is NU3042, then we shouldn't need it.

I'm not sure I understand you.  The URL is `https://aka.ms/nuget/nu3042` not `NU3042`.  The idea here is that there is a clear next step (follow the given URL).